addons/website/static/src/snippets/s_numbers_showcase/000.scss

```
@@ -0,0 +1,4 @@
+.s_numbers_showcase:not([data-vcss]) .row > div {
```

Nitpicking: no need the increase the selector specificity until a `001.scss` will be created ;)

```suggestion
.s_numbers_showcase .row > div {
```